test/powershell/Language/Scripting/NativeExecution/NativeCommandProcessor.Tests.ps1

```diff
@@ -72,7 +72,7 @@ Describe "Native Command Processor" -tags "Feature" {
                 break
```

Comment:
I didn't make changes to the test. I think if you have concerns about the test, we should handle that as a separate issue and PR. I'm only trying to converge to a single test executable.